gemfiles/rails_5_2.gemfile

```diff
@@ -4,14 +4,14 @@ source "https://rubygems.org"
 
 gem "rails", "~> 5.2"
 gem "railties", "~> 5.2"
-gem "capybara", "~> 3.0"
+gem "capybara", "~> 3"
```

Comment:
This actually does nothing, and is kind of a bug, so we should just leave this as is (The new styling would allow any version of capybara)